homeassistant/components/shelly/coordinator.py

```diff
@@ -154,24 +154,27 @@ def async_setup(self, pending_platforms: list[Platform] | None = None) -> None:
         )
         self.device_id = device_entry.id
 
-    async def _async_device_connect(self) -> None:
-        """Connect to a Shelly Block device."""
+    async def _async_device_connect_task(self) -> bool:
+        """Connect to a Shelly device task."""
         LOGGER.debug("Connecting to Shelly Device - %s", self.name)
         try:
             await self.device.initialize()
             update_device_fw_info(self.hass, self.device, self.entry)
         except DeviceConnectionError as err:
-            raise UpdateFailed(f"Device disconnected: {repr(err)}") from err
+            LOGGER.error(
```

Comment:
Is this going to log every time it retries?

Reply:
Yes, that is not good, maybe changing level to debug will be better?